config/ground-truth-yaml-files/all_features_version_10.1_no_visit_variant_covid.yml

```diff
@@ -549,306 +475,316 @@ patient:
-    - limit: 1
-      search_term: carbon monoxide
+    - limit: 10
+      search_term: carbon monoxide exposure
+    type: integer
   AvgDailyFormaldehydeExposure_2:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: formaldehyde
+    - limit: 10
+      search_term: formaldehyde exposure
+    type: integer
   AvgDailyFormaldehydeExposure_2_qcut:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: formaldehyde
+    - limit: 10
+      search_term: formaldehyde exposure
+    type: integer
   AvgDailyNO2Exposure_2:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: nitrogen dioxide
+    - limit: 10
+      search_term: nitrogen dioxide exposure
+    type: integer
   AvgDailyNO2Exposure_2_qcut:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: nitrogen dioxide
+    - limit: 10
+      search_term: nitrogen dioxide exposure
+    type: integer
   AvgDailyNOExposure_2:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: nitric oxide
+    - limit: 10
+      search_term: nitric oxide exposure
+    type: integer
   AvgDailyNOExposure_2_qcut:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: nitric oxide
+    - limit: 10
+      search_term: nitric oxide exposure
+    type: integer
   AvgDailyNOxExposure_2:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
     - limit: 1
       search_term: nitrogen oxides
+    type: integer
   AvgDailyNOxExposure_2_qcut:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
     - limit: 1
       search_term: nitrogen oxides
+    type: integer
   AvgDailyOzoneExposure:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
     minimum: 1
-    type: integer
     name_lookup:
-    - limit: 1
-      search_term: ozone
+    - limit: 10
+      search_term: ozone exposure
+    type: integer
   AvgDailyOzoneExposure_StudyAvg:
     categories:
     - biolink:ChemicalEntity
     - biolink:EnvironmentalExposure
     maximum: 5
-    minimum: 1
-    type: integer
+    minimum: 10

```

Comment:
Should the name_lookup be 10 and this be 1?

Reply:
Fixed and committed to branch.

Reply:
I fixed an error related to one of the age variables and committed to branch. However, your Slack message suggests that I misunderstood your comment. Perhaps you can explain? Thanks!

Reply:
Oh, I see. I set the minimum to 1 and kept the maximum at 5. Committed to branch.